nimbus/src/client/http_client.rs

```diff
@@ -148,6 +203,7 @@ mod tests {
         assert_eq!(
             exp.clone(),
             Experiment {
+                schema_version: "1.0.0".to_string(),
```

Comment:
nit: since you're hard-coding the expected schema version here in the assertion, I think it would also be fine to hard-code the schema versions in the test data rather than formatting based on `SCHEMA_VERSION`. Alternately, this should also check for `format!("{}.0.0", SCHEMA_VERSION)` rather than hard-coding, for consistency.